frigate/events/cleanup.py

```diff
@@ -230,7 +230,15 @@ def run(self) -> None:
                     Event.delete().where(Event.id << chunk).execute()
 
                     if self.config.semantic_search.enabled:
-                        self.embeddings.thumbnail.delete(ids=chunk)
-                        self.embeddings.description.delete(ids=chunk)
+                        for collection in [
+                            self.embeddings.thumbnail,
+                            self.embeddings.description,
+                        ]:
+                            existing_ids = collection.get(ids=chunk, include=[])["ids"]
```

Comment:
Seems like a lot of extra db activity to read before deleting, maybe there's a better way to avoid the log?

Reply:
We're not fetching any data itself, just the ids. So it's fast